src/backend/drive/settings.py

```diff
@@ -308,6 +309,7 @@ class Base(Configuration):
     # Django applications from the highest priority to the lowest
     INSTALLED_APPS = [
         "core",
+        "demo",
```

Comment:
Should be added only in the `Test` and `Development` config. Not in the base one.